apps/mull-ui/src/app/app.tsx

```diff
@@ -50,7 +50,6 @@ export const App = () => {
               <Route path={ROUTES.DISCOVER} component={() => <div>DISCOVER!</div>} />
               <Route path={ROUTES.UPCOMING} component={() => <div>UPCOMING!</div>} />
               <Route path={ROUTES.MY_EVENTS} component={() => <div>MY_EVENTS!</div>} />
-              <Route path={ROUTES.HOME} render={() => <Redirect to={ROUTES.DISCOVER} />} />
```

Comment:
This was a suggestion I gave to Ragith to redirect to `/home/discover` if the user goes to `/home/somethingelse`, however, this seems to break the SwipeableRoutes slightly, so I decided to remove it for now

Reply:
Feel free to make a bug report about `/home/somepage` I can fix it

Reply:
Ok ill remove it from this pr